### ISSUE TITLE
[REQ] Graph request

### ISSUE THREAD

**Issue by markonline1** [cagnulein/qdomyos-zwift#256]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**
As mentioned on the Facebook page, the final graph that currently shows Resistance and cadence in the beta, could we get power and HR added so the final graph shows cadence, resistance, power and HR? Thanks.

**Comment by cagnulein** [comment]
Ok probably i need to add a new setting in order to activate this. What do
you think?

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno gio 8 apr 2021 alle ore 20:29 markonline1 <
***@***.***> ha scritto:

**Comment by cagnulein** [comment]
Yep, the problem is that a chart needs time to be loaded, so i will have to
allow the user to select which chart show. But it's totally doable! :)

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno gio 8 apr 2021 alle ore 21:11 markonline1 <
***@***.***> ha scritto:

> Hi Robert,
>
> I’m not looking to make more work for you, you must already be putting a
> load of work into this already. If it’s a fairly easy fix though, I think
> it would be a benefit. Strava currently produces the graphs QZ will
> (although I realize not everyone uses Strava) and the peloton app produces
> a HR and cadence graph (again, not everyone connects to the peloton app).
> But I’m not aware of anywhere that produces all the important data in one
> graph which I why I think it would be cool and beneficial to have
> everything. Maybe the option to choose what is displayed on the final graph
> with sliders so people can choose what is displayed?
>
> Thanks
>
> Mark
>
> Sent from my iPhone
>

**Comment by markonline1** [comment]
Ok. I haven’t used the beta so haven’t had a chance to try the charts, I’ve only seen the screen grabs. If you think it’s feasible, great, if it’s not, totally understand. I think ultimately you know what’s best for the app, just chucking out ideas. Either way, I look forward to seeing how you keep developing this app, as it’s obvious you seem keen on making continuous improvements to an already great product. 
Cheers

Mark

**Comment by cagnulein** [comment]
@markonline1 are you happy with the current implementation?
